### PR TITLE
1.29.2 fix pg vulnerability

### PR DESCRIPTION
**Pull request by oleurud**
This PR updated the old 1.29.2 version with:
- Adding a new cartodb-psql fixing the pg vulnerability
- Added some changes to pass test (upgrading old versions or old assets difficult to find today)

This will be the tag 1.29.3

-----------------

More info about pg vulnerability 
https://node-postgres.com/announcements#2017-08-12-code-execution-vulnerability

### CODE REVIEW

**Comment by oleurud** [comment]
I have changed the package.js updating the cartodb-psql to "~0.6.0"

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+1.29.3 - 2017-08-31
```
Comment (dgaubert) [inline NEWS.md]: I'd wait to set the date after merge and before creating the new tag. No need to change it, just for heads up.

File: test/prepare_db.sh [hunk 2]
```diff
@@ -85,7 +85,7 @@ if test x"$PREPARE_PGSQL" = xyes; then
 
   # TODO: send in a single run, togheter with test.sql
   psql -c "CREATE EXTENSION plpythonu;" ${TEST_DB}
-  for i in CDB_QueryStatements CDB_QueryTables CDB_CartodbfyTable CDB_TableMetadata CDB_ForeignTable CDB_UserTables CDB_ColumnNames CDB_ZoomFromScale CDB_Overviews
+  for i in CDB_QueryStatements CDB_QueryTables CDB_CartodbfyTable CDB_TableMetadata CDB_ForeignTable CDB_UserTables CDB_ColumnNames CDB_ZoomFromScale CDB_OverviewsSupport CDB_Overviews
```
Comment (dgaubert) [inline test/prepare_db.sh]: question, why do we need to add `CDB_OverviewsSupport`? the new version of `cartodb-psql` is using it?

Comment (oleurud) [reply test/prepare_db.sh]: The CDB_Overviews uses a function created by CDB_OverviewsSupport (_CDB_UserTablesInSchema)

How this files are not versioned, we always use the same files. I suppose that in the past, when the version 1.29.2 was created, or CDB_Overviews did not use _CDB_UserTablesInSchema or the function was created in the CDB_Overviews file

Comment (dgaubert) [reply test/prepare_db.sh]: Oh! I see, thanks for the clarification!

**Comment by dgaubert** [comment]
🇪🇸 🇨🇭

**Comment by oleurud** [comment]
The tag 1.29.3 was created

**Pull request merged**